docs/reference/builder.md

```diff
@@ -754,14 +754,15 @@ This will then be visible from `docker inspect` with the other labels.
 
 ## EXPOSE
 
-    EXPOSE <port> [<port>...]
+    EXPOSE <port> [<port>/<protocol>...]
 
 The `EXPOSE` instruction informs Docker that the container listens on the
-specified network ports at runtime. `EXPOSE` does not make the ports of the
-container accessible to the host. To do that, you must use either the `-p` flag
-to publish a range of ports or the `-P` flag to publish all of the exposed
-ports. You can expose one port number and publish it externally under another
-number.
+specified network ports at runtime. You can also specify if the port uses TCP/UDP,
+and defaults to tcp if the protocol is omitted.
+`EXPOSE` does not make the ports of the container accessible to the host. 
```

Comment:
I was then thinking this, "`EXPOSE` does not make the ports of the specified network to the host ports at runtime, instead it allows for users running the container to be aware what ports are available in the container". 
Also, I forgot to mention that I made this pull request when I saw this [issue](https://github.com/moby/moby/issues/34373), is there any problem? Sorry, I'm quite new to this.